cli/providers/aws/translate.go

```diff
@@ -334,7 +335,7 @@ func loadIAM(t *gocf.Template, s *service.Service, e *environment.Environment) {
 			"Statement": map[string]interface{}{
 				"Effect":   "Allow",
 				"Action":   []string{"logs:CreateLogStream", "logs:PutLogEvents"},
-				"Resource": fmt.Sprintf("arn:aws:logs:%s:*:log-group:i2kit-%s:log-stream:i-*", e.Provider.Region, s.GetFullName(e, "-")),
+				"Resource": fmt.Sprintf("arn:aws:logs:%s:*:log-group:i2kit-%s:log-stream:*", e.Provider.Region, s.GetFullName(e, "-")),
```

Comment:
would it be possible to customize the "i2kit" string to a value send by the caller?